tests/regression/lp-1848567/task.yaml

```diff
@@ -20,7 +20,7 @@ prepare: |
   snap connect test-snapd-app:sound-themes test-snapd-gtk-common-themes:sound-themes
 
 execute: |
-  if snap debug sandbox-features --required apparmor:kernel:mount; then
+  if snap debug sandbox-features --required apparmor:kernel:mount && ! snap debug sandbox-features --required parser:snapd-internal; then
```

Comment:
This should be `apparmor:parser:snapd-internal`

Reply:
Thanks!